packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -118,12 +118,14 @@ export function Autocomplete({
         onFocus={handleInputFocus}
         onBlur={handleInputBlur}
       />
-      <Menu
-        visible={menuVisible}
-        options={options}
-        selectedOption={value}
-        onOptionSelect={handleMenuChange}
-      />
+      {menuVisible && (
+        <Menu
+          visible={true}
```

Comment:
I don't disagree with this change. Are we doing this to make sure Menu isn't getting rendered when it is invisible?

Reply:
yep, this basically takes the visibility from css into react